nodejs/week1/homework/src/backend/fetchData.js

```diff
@@ -0,0 +1,28 @@
+const reviews = require('./data/reviews');
+const meals = require('./data/meals'); //{ ...obj, reviews: [] }
+
+//  add reviews to all meals and empty to what meal without reviews
+const reviewsInMeals = meals.map(meal => ({
+  ...meal,
+  reviews: findReviewsForMeals(meal.id),
+}));
+
+function findReviewsForMeals(mealID) {
```

Comment:
It is not necessary to create a function for this, since you are only using it once! But is it of course not wrong :)